src/mon/OSDMonitor.cc

```diff
@@ -11316,6 +11319,22 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
 	}
       }
 
+      string technique = profile_map["technique"];
+      if (technique == "blaum_roth") {
+	bool experimental_enabled =
+	  g_ceph_context->check_experimental_feature_enabled("blaum_roth");
```

Comment:
Even if we're only actually checking blaum_roth, let's make this one experimental_ec_profiles.

Reply:
Got it, will make that change.

Reply:
Fixed.